util/feed.go

```diff
@@ -91,6 +91,9 @@ type Feed struct {
 // all Subscribers to the feed. Events published to a closed feed, or to a feed
 // with no Subscribers, will be ignored.
 func (f *Feed) Publish(event interface{}) {
+	if f == nil {
```

Comment:
I misread this diff, because I was 100% positive I had added this behavior before and it highlighted because it was being taken out.  Reading it now, it's clear that its being added; my brain is clearly broken.  

The nil behavior that you have added is the correct thing to do, disregard my comment.